csrc/scheduler/registry.cpp

```diff
@@ -513,6 +513,22 @@ class SchedulerTopologyChecker {
   }
 };
 
+// we only block merging of SegmenterLoad
```

Comment:
Does this mean if a fusion has `SegemterLoad`, the fusion outputs must only have the output of the `SegmenterLoad`?

Reply:
Sorry about the poor documenting. This is a mess since I wasn't clear about the semantics and ended up with conflicting names all over the place.

I renamed it to `SegmenterSet` instead. but the logic here will be the same.

When you have a `SegmenterSet`, it indicates that kernel will be broken up on the output of the op. i.e. Each `SegmenterSet` op's output should be a fusion output.

This is assuming that there's only one consumer of that. I think I shouldn't have that and instead I should change the logic to check that the output of `SegmenterSet` isn't consumed by any node in the fusion.